tsconfig.json

```diff
@@ -20,6 +20,6 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src/**/*.ts", "src/**/*.d.ts", "src/**/*.vue"],
```

Comment:
This change surprised me. I thought we just wanted to remove type checking from spec files?

Reply:
Well we don't have any .vue files and *.ts includes *.d.ts. So idk that anything here changed. I was just cleaning up cause I came to look at the tsconfig.

Reply:
I didn't actually remove the .spec files from the tsconfig (in order to do that we'd have to convert them to .js). Just removed source checking from the type testing command